apps/challenges/views.py

```diff
@@ -1827,8 +1828,8 @@ def download_all_submissions(
                             submission["created_at"],
                             submission["submission_result_file"],
                             submission["submission_metadata_file"],
-                            submission["method_name"],
-                            submission["method_description"],
+                            "".join("-" if c in conversion else c for c in submission["method_name"]),
```

Comment:
@RishabhJain2018 why can't we use `.replace` method?

Reply:
yes, make sense. I was testing with a bunch of characters and finally, it reduced to one.